superbench/runner/runner.py

```diff
@@ -116,6 +117,9 @@ def __get_mode_command(self, benchmark_name, mode):
             name=benchmark_name,
             output_dir=self._sb_output_dir,
         )
+        if timeout is not None:
+            exec_command = 'timeout {timeout} {command}'.format(timeout=timeout, command=exec_command)
```

Comment:
do we still need the bash timeout if you add it in ansible?

Reply:
does it have some side effects? e.g., cannot pass env correctly

Reply:
I tested that ansible timeout only early stop the server side, 'sb exec' processes are still running on client side. We need to terminate all the processes manually.
Didn't find the pass env issue, tested some E2E model benchmarks and microbenchmarks, it works well.